src/style/properties.js

```diff
@@ -33,6 +33,7 @@ var styfn = {};
     bidirectionalSize: { number: true }, // allows negative
     bidirectionalSizes: { number: true, multiple: true }, // allows negative
     sizeMaybePercent: { number: true, min: 0, allowPercent: true },
+    paddingRelativeTo: { enums: [ 'width', 'height', 'whAverage', 'whMin', 'whMax' ] },
```

Comment:
Style property values should always be `dash-case` rather than `camelCase`.

I think `width, height, average, min, max` is fine